src/server.cpp

```diff
@@ -318,6 +321,22 @@ int Server<IoType, SwitchActivityInfo, SwitchCalcGaps>::server_accept(int ifd) {
                                         inet_ntoa(addr.sin_addr), ntohs(addr.sin_port), active_ifd);
                             }
                             do_accept = true;
+
+                            if (s_user_params.sockperfDumpDataToFile)
+                            {
+                                retVal = system((createSockperfDirectoryCommand.append(s_user_params.sockperfDumpDataDirectory)).c_str());
```

Comment:
Why not to create using the mkdir function?https://codeyarns.com/2014/08/07/how-to-create-directory-using-c-on-linux/